tekton/resources/nightly-tests/bastion-z/cleanup_tekton.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cleanup-tekton-nightly
+spec:
+  params:
+  - name: package
+  - name: resources
+    description: space separated list of resources to be deleted
+    default: "conditions pipelineresources tasks pipelines taskruns pipelineruns"
+  - name: kubeconfig-secret
```

Comment:
You are using workspaces for the secret (yay), so you don't need this parameter anymore.
You could instead have a parameter to specify the name of the key within the secret (which is hardcoded to `config` now, in L30 below).

Reply:
workspace for ssh key is now used